documentation/user_docs/docs/manual/Correcting_for_sample_misalignment.rst

```diff
@@ -193,72 +226,145 @@ The keyword options are:
+that adding new peaks would not improve accuracy of your alignment, and that the resulting alignment is satisfactory.
+In other words, you are following :ref:`the iterative process<StepA>` above.
+You want to get your results quickly and possibly experiment with them, modify them and apply or undo your a quickly. 
+In this case you apply correctios to existing ``sqw`` file or ``sqw`` object loaded in memory.
 
-- When you are still accumulating data (e.g. on the beamline):
+2. When you are satisfied with the result of alignment you may want to regenerate your ``sqw`` file after calculating goniometer
+offsets, which define actual crystal position. You have to do this step if you want to apply various symmetry 
+transformations to the whole ``sqw`` file during generation. Alternatively, you may want to "finalize" alignment corrections
+applied initially.
 
-  Calculate what the goniometer offsets for regeneration
+Both ways result in an sqw file; the resulting files are identical from a physical point of view.
+Minor differences occurs in the data, stored in an sqw file.
 
 
-Option 1 : apply the correction to an existing sqw file
-=======================================================
+Option 1 : apply the correction to an existing sqw file or object
+-----------------------------------------------------------------
 
-There is a simple routine to apply the changes to an existing file, without the need to regenerate it from raw data
+There is a simple and fast routine ``change_crystal`` to apply the changes to an existing file, without the need to regenerate it from raw data.
 
 ::
 
-   change_crystal(win, alignment_info)
+   >>change_crystal(win, alignment_info);
+   or 
+   >>wout = change_crystal(win, alignment_info);
+
+The second form of this routine returns aligned ``sqw`` object. The object is filebacked if pixels data are too big to be loaded in memory.
+The second form is mandatory if you are applying alignment to ``sqw`` object in memory.
 
-where ``alignment_info`` was determined in the steps described above. This procedure modifies lattice parameters and 
-the pixels will be aligned whenever they are loaded or manipulated (e.g. accessing pixel data, cutting, plotting, etc.).
+Here ``win`` is misaligned ``sqw`` file or ``sqw`` object and ``alignment_info`` was determined on the :ref:`Step 2<Step_2_misalignment_correction>` described above.
 
-Once you have confirmed that the alignment you have is the correct one, it is possible to fix the alignment to avoid this calculation step.
+Majority of Horace users may work with files realigned using ``change_crystal`` without any noticeable hindrance. 
+Advanced users may want to know, that ``change_crystal`` procedure modifies lattice parameters and adds alignment matrix to the pixels data.
+Pixels themselves are not modified so the alignment procedure is very fast. Pixels will be aligned whenever they are loaded or manipulated 
+(e.g. accessing pixel data, cutting, plotting, etc.). 
```

Comment:
Thanks for splitting up the lines in the markdown like this. It makes it easier to read the large mass of text.
For improving the display in the processed text, a single blank line here after line 281 splits the text into two paragraphs, each commenting on one aspect of the revised code performance.

Reply:
not sure about empty line or after 281, but text intended for normal users and advanced users can certanly be ceparated into to paragraphs.